packages/honeypot/tsconfig.cjs.json

```diff
@@ -0,0 +1,15 @@
+{
+    "extends": "../../tsconfig.cjs.json",
+    "compilerOptions": {
+        "rootDir": "./src",
+        "outDir": "./dist/cjs",
+        "lib": ["es6", "dom"],
+        "jsxImportSource": "@emotion/react"
+    },
+    "include": ["src/index.tsx"],
+    "references": [
+        {
+            "path": "../util/tsconfig.cjs.json"
```

Comment:
Is this required? You have no imports from @prosopo/util in your package.

Reply:
Sorry, I didn't removed it after test
I will remove these